src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1531,6 +1531,17 @@ CLParser::VerifyOptions()
   if (params_->metrics_interval_ms == 0) {
     Usage("Metrics interval must be larger than 0 milliseconds.");
   }
+
+  if (params_->should_collect_metrics && !params_->metrics_url_specified) {
```

Comment:
should this if be gated by params->url_specified as well?
This way we aren't doing work in the default case?

Reply:
discussed offline